src/frontend/AboutDialog.ui

```diff
@@ -1,869 +1,2568 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
![too_much_about](https://user-images.githubusercontent.com/9459891/43120616-ab9ebc3e-8f3c-11e8-84b0-63f822b4b883.png)
Too much spaces!

Reply:
This is OLD School one, I'm not gonna change it's layout for now. It requires some rework.